src/sentry/types/issues.py

```diff
@@ -25,6 +26,7 @@ class GroupCategory(Enum):
     GroupType.PERFORMANCE_LONG_TASK_SPANS: GroupCategory.PERFORMANCE,
     GroupType.PERFORMANCE_RENDER_BLOCKING_ASSET_SPAN: GroupCategory.PERFORMANCE,
     GroupType.PERFORMANCE_DUPLICATE_SPANS: GroupCategory.PERFORMANCE,
+    GroupType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES: GroupCategory.PERFORMANCE,
```

Comment:
oh it'll also need to be added here https://github.com/getsentry/sentry/blob/master/src/sentry/models/group.py#L405-L409

Reply:
Great catch, thanks! Done

Reply:
🙏